### ISSUE TITLE
initial weights issue

### ISSUE THREAD

**Issue by mama110** [pytorch/pytorch#15658]
If I define conv2d like this:

        self.Conv1 = [
            torch.nn.Conv2d(3, 32, kernel_size=3, stride=1, padding=1),  
            torch.nn.Conv2d(32, 64, kernel_size=3, stride=2, padding=1) 
        ]
        .......
        self.Classfier = [
            torch.nn.Linear(13*13*1024, 1000) 
        ]

How can I initial the weights?

**Comment by vishwakftw** [comment]
Please look at the answer here:
https://discuss.pytorch.org/t/custom-weight-initialization/20544

These questions will be attended to faster in the [forums](https://discuss.pytorch.org), so please post there.
